Welthungerhilfe/cgm-rg
# Build and test pipeline

Pipeline to perform the following task

Steps: 

1. Build the container 
2. Run the container with pytest
3. Copy the test result
4. Publish the test result

This will ensure the code is properly tested and before getting merged to develop branch
